editor/src/clj/editor/welcome.clj

```diff
@@ -408,11 +408,20 @@
           (.setFixedCellSize 56.0)
           (ui/items! (recent-projects prefs))
           (ui/cell-factory!
-            (fn [recent-project]
-              {:graphic (make-recent-project-entry recent-project recent-projects-list prefs localization)}))
+           (fn [recent-project]
+             {:graphic (make-recent-project-entry recent-project recent-projects-list prefs localization)}))
```

Comment:
Style: The indentation changed on these two lines, but it was correct before. We use two spaces to indent function arguments on their own line.
```suggestion
            (fn [recent-project]
              {:graphic (make-recent-project-entry recent-project recent-projects-list prefs localization)}))
```